docs/react/introduce.en-US.md

```diff
@@ -0,0 +1,95 @@
+---
+order: 0
+title: Ant Design of React
+---
+
+antd is a set of React components which follow Ant Design specification. It is designed to develop RIA sucn as dashboard to serve enterprise.
+
+<div class="pic-plus">
+  <img width="150" src="https://t.alipayobjects.com/images/rmsweb/T11aVgXc4eXXXXXXXX.svg">
+  <span>+</span>
+  <img width="160" src="https://t.alipayobjects.com/images/rmsweb/T16xRhXkxbXXXXXXXX.svg">
+</div>
+
+<style>
+.pic-plus > * {
+  display: inline-block!important;
+  vertical-align: middle;
+}
+.pic-plus span {
+  font-size: 30px;
+  color: #aaa;
+  margin: 0 20px;
+}
+</style>
+
+---
+
+## Features
+
+- Designed as Ant Design, a design language to create user friendly and beautiful websites.
+- It is a set of high quality UI components and based on [React Component](http://react-component.github.io/badgeboard/).
+- Provide a work flow which is based on npm and webpack and babel, it supports ES2015 and TypeScript.
+
+## Installation
+
+```bash
+$ npm install antd
+```
+
+## Example
+
+```jsx
+import { DatePicker } from 'antd';
+ReactDOM.render(<DatePicker />, mountNode);
+```
+
+Import style:
+
+```jsx
+import 'antd/dist/antd.css';  // or 'antd/dist/antd.less'
+```
+
+You can use `import DatePicker from 'antd/lib/date-picker'` or [babel-plugin-antd](https://github.com/ant-design/babel-plugin-antd)(This plugin supports import component and styles on demand) to import components on demand.
+
+## Version
+
+- Stable: [![npm package](http://img.shields.io/npm/v/antd.svg?style=flat-square)](https://www.npmjs.org/package/antd)
+- Beta: [![](https://cnpmjs.org/badge/v/antd.svg?&tag=beta&subject=npm)](https://www.npmjs.org/package/antd)
+
+## Compatibility
+
+Modern browsers and IE8+.
+
+> [IE8 issues](https://github.com/xcatliu/react-ie8)
+
+## Useful Links
+
+- [Home Page](http://ant.design/)
+- [Changelog](/changelog)
+- [Scaffold](https://github.com/ant-design/antd-init/)
+- [Develop Tools](http://ant-tool.github.io/)
+- [React Component](http://react-component.github.io/)
+- [Ant Design Mobile](http://mobile.ant.design)
+- [React Code Style](https://github.com/react-component/react-component.github.io/blob/master/docs/zh-cn/component-code-style.md)
+- [Design Principle of Component](https://github.com/react-component/react-component.github.io/blob/master/docs/zh-cn/component-design.md)
```

Comment:
- [Design Principles of Component](https://github.com/react-component/react-component.github.io/blob/master/docs/en-us/component-design.md)

TODO: @benjycui This also needs to be translated, but the link should point to this near-future-available version.